source/Sashimi.AzureWebApp.Tests/DeployAzureWebCommandFixture.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Calamari.AzureWebApp;
+using Calamari.Common.Features.Deployment;
+using Calamari.Common.Features.Scripts;
+using Calamari.Common.Plumbing.FileSystem;
+using Calamari.Tests.Shared;
+using FluentAssertions;
+using Microsoft.Azure.Management.AppService.Fluent;
+using Microsoft.Azure.Management.Fluent;
+using Microsoft.Azure.Management.ResourceManager.Fluent;
+using Microsoft.Azure.Management.ResourceManager.Fluent.Core;
+using NUnit.Framework;
+using Sashimi.Tests.Shared;
+using Sashimi.Tests.Shared.Server;
+using KnownVariables = Calamari.Common.Plumbing.Variables.KnownVariables;
+using OperatingSystem = Microsoft.Azure.Management.AppService.Fluent.OperatingSystem;
+
+namespace Sashimi.AzureWebApp.Tests
+{
+    [TestFixture]
+    class DeployAzureWebCommandFixture
+    {
+        IAppServicePlan appServicePlan;
+        IResourceGroup resourceGroup;
+        IAzure azure;
+        string clientId;
+        string clientSecret;
+        string tenantId;
+        string subscriptionId;
+
+        readonly HttpClient client = new HttpClient();
+        TemporaryDirectory azureConfigPath;
+
+        [OneTimeSetUp]
+        public async Task Setup()
+        {
+            azureConfigPath = TemporaryDirectory.Create();
+            Environment.SetEnvironmentVariable("AZURE_CONFIG_DIR", azureConfigPath.DirectoryPath);
```

Comment:
This ensures that we are not using the current users Azure context, hence the Azure context is not authenticated